src/drivers/display/drm/lv_linux_drm.c

```diff
@@ -109,6 +112,13 @@ static void drm_flush(lv_display_t * disp, const lv_area_t * area, uint8_t * px_
 
 lv_display_t * lv_linux_drm_create(void)
 {
+    static bool inited = false;
+
+    if(!inited) {
```

Comment:
No judgment is required, and it does not matter if the settings are repeated.

Reply:
Fixed